contrib/http_base/src/main/java/io/opentelemetry/contrib/http/core/StatusCodeConverter.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.contrib.http.core;
+
+import io.opentelemetry.trace.Status;
+import javax.annotation.Nullable;
+
+/** Provides standard translation from HTTP status codes to Open Telemetry trace status values. */
+public class StatusCodeConverter {
+
+  /**
+   * Translates the supplied HTTP status code to equivalent Open Telemetry status.
+   *
+   * @param httpStatus the source value
+   * @return the target value
+   */
+  public Status convert(int httpStatus) {
+    Status status = doCustomConvert(httpStatus);
+    if (status != null) {
+      return status;
+    }
+    if (httpStatus >= 200 && httpStatus < 400) {
+      status = Status.OK;
+    } else if (httpStatus == 401) {
```

Comment:
rather than this gigantic stack of if/elses, put the specific status codes into a Map.  Also, you should return early when you have an answer, rather than have all the elses.

Reply:
Done